ui/src/app/views/workspace-share/component.html

```diff
@@ -9,26 +9,31 @@ <h3 class="modal-title-text">Share {{workspace.name}}</h3>
       workspace with them.</p>
     <div class="sharing-area">
       <div class="adding-area">
-        <div class="add-by-text-area">
-          <label class="text-label" class="collaborator-label">Find Collaborators</label>
-          <input type="text" class="input share" name="to-share" autofocus
-              [(ngModel)]="toShare" placeholder="enter {{gsuiteDomain}} ID" (ngModelChange)="userNotFound = false"
-              #usernameSharingInput [disabled]="!hasPermission">
-          <button type="button" class="btn btn-primary sharing-buttons add-button" (click)="addCollaborator()" [disabled]="!hasPermission">Add</button>
-          <div>
-            <clr-dropdown>
-              <button type="button" class="btn btn-outline-primary permissions-button" clrDropdownTrigger [disabled]="!hasPermission">
-                {{selectedPermission}}
-                <clr-icon shape="caret down"></clr-icon>
-              </button>
-              <clr-dropdown-menu *clrIfOpen>
-                <button type="button" clrDropdownItem (click)="setAccess('Reader')">Reader</button>
-                <button type="button" clrDropdownItem (click)="setAccess('Writer')">Writer</button>
-                <button type="button" clrDropdownItem (click)="setAccess('Owner')">Owner</button>
-              </clr-dropdown-menu>
-            </clr-dropdown>
+        <clr-tooltip>
+          <div class="add-by-text-area" clrTooltipTrigger="!hasPermission">
+            <label class="text-label" class="collaborator-label">Find Collaborators</label>
+            <input type="text" class="input share" name="to-share" autofocus
+                [(ngModel)]="toShare" placeholder="enter {{gsuiteDomain}} ID" (ngModelChange)="userNotFound = false"
+                #usernameSharingInput [disabled]="!hasPermission">
+            <button type="button" class="btn btn-primary sharing-buttons add-button" (click)="addCollaborator()" [disabled]="!hasPermission">Add</button>
+            <div>
+              <clr-dropdown>
+                <button type="button" class="btn btn-outline-primary permissions-button" clrDropdownTrigger [disabled]="!hasPermission">
+                  {{selectedPermission}}
+                  <clr-icon shape="caret down"></clr-icon>
+                </button>
+                <clr-dropdown-menu *clrIfOpen>
+                  <button type="button" clrDropdownItem (click)="setAccess('Reader')">Reader</button>
+                  <button type="button" clrDropdownItem (click)="setAccess('Writer')">Writer</button>
+                  <button type="button" clrDropdownItem (click)="setAccess('Owner')">Owner</button>
+                </clr-dropdown-menu>
+              </clr-dropdown>
+            </div>
           </div>
-        </div>
+          <clr-tooltip-content *clrIfOpen clrPosition="left" clrSize="md">
+            Owner access is required to share the workspace.
```

Comment:
Any reason not to just show this inline? (People may not think to hover over the text area.)